CHANGELOG.rst

```diff
@@ -2,6 +2,16 @@
 Changelog
 =========
 
+0.14.0 (YYYY-MM-DD)
+-------------------
+
+* Now requires ``packaging >= 20.0``.
+* Fixed detection of the :class:`~.Returns` base class.
+* Improved docs.
+* Dropped Python 3.7 support.
```

Comment:
Maybe move this higher? I would consider making it the first entry, but if nothing else I would put it above doc improvements.